--- a/src/Application.vala	2019-05-28 17:54:52.000000000 +0200
+++ b/src/Application.vala	2019-10-11 23:13:11.572515910 +0200
@@ -33,6 +33,9 @@
     public override void activate () {
         var window = new MainWindow (this);
 
+        Gtk.Settings.get_default().set_property("gtk-theme-name", "elementary");
+        Gtk.Settings.get_default().set_property("gtk-icon-theme-name", "elementary");
+
         var quit_action = new SimpleAction ("quit", null);
         add_action (quit_action);
         set_accels_for_action ("app.quit", {"<Control>q"});
